client/packages/programs/src/JsonForms/common/components/CategorizationTabLayout.tsx

```diff
@@ -95,6 +99,75 @@ const CategoryLayoutRendererComponent = ({
 };
 const CategoryLayoutRenderer = React.memo(CategoryLayoutRendererComponent);
 
+// Try to extract a more precise error path
+const propertyPathFromError = (error: {
```

Comment:
Perhaps these two methods would be useful in a `utils` file? They look like useful shared functionality

Reply:
I leave it for another time when I really needed it somewhere else (it was a bit hacky and took longer than I wanted so would like to give it a second thought then...)